### DISCUSSION TITLE
Selected item bigger than unselected items

### DISCUSSION THREAD

**Discussion by Anderzzon** [SplashByte/animated_toggle_switch#63] [Q&A] [answered]
![Screenshot 2024-11-05 at 09 49 37](https://github.com/user-attachments/assets/018887b8-62bd-4f51-8c61-7ec476baec00)
![Screenshot 2024-11-05 at 09 49 29](https://github.com/user-attachments/assets/9e7b2908-8ce3-43bc-81d3-9dd49dc0907e)
I've got this from our designer but are not able to figure out if this is possible with this library. 
The selected item should have 10px boarder making it look larger than the unselected item.

**Comment by maeddin** [comment, marked as answer]
Currently you cannot implement this with `AnimatedToggleSwitch` (I guess). However, you can do it with `CustomAnimatedToggleSwitch`. Here you can set everything, but you also have to implement more yourself, as more parameters are required. If you get stuck with the implementation, you are welcome to ask here.

Would that be a solution for you? Otherwise I would first have to see how to implement the feature in `AnimatedToggleSwitch` without breaking changes.

**Reply by maeddin** [reply]
@Anderzzon Could I help you with this or does `CustomAnimatedToggleSwitch` not work for you?

**Reply by Anderzzon** [reply]
Thank you for the help!
I ended up simplifying the designers design a little bit so I could use  AnimatedToggleSwitch. If I get time later I might look into CustomAnimatedToggleSwitch and try to implement the original design.
